File: src/main/java/cn/mobiledaily/domain/Exhibitor.java
package cn.mobiledaily.domain;

/**
 * Created with IntelliJ IDEA.
 * User: Johnson
 * Date: 3/4/13
 * Time: 1:17 AM
 * To change this template use File | Settings | File Templates.
 */
public class Exhibitor {
    private String name;
    private String stand;
    private String company;
    private String email;
    private String address;
    private String website;
    private String category;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getStand() {
        return stand;
    }

    public void setStand(String stand) {
        this.stand = stand;
    }

    public String getCompany() {
        return company;
    }

    public void setCompany(String company) {
        this.company = company;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getWebsite() {
        return website;
    }

    public void setWebsite(String website) {
        this.website = website;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }
}

File: src/main/scala/cn/mobiledaily/domain/Exhibitor.scala
package cn.mobiledaily.domain

import org.springframework.data.annotation.Transient
import cn.mobiledaily.domain.mobile.ExhibitionContent

class Exhibitor extends Domain with ExhibitionContent {
    def getCollectionSuffix: String = ".exhibitor"

    @Transient
    var exhibition: Exhibition = null
    var name: String = null
    var stand: String = null
    var company: String = null
    var email: String = null
    var address: String = null
    var website: String = null
    var category: String = null
    var location: String = null
    var phone: String = null
    var description: String = null

    def getExhibition = exhibition
    def getName = name
    def getStand = stand
    def getCompany = company
    def getEmail = email
    def getAddress = address
    def getWebsite = website
    def getCategory = category
    def getLocation = location
    def getPhone = phone
    def getDescription = description

    def setExhibition(v: Exhibition) {exhibition = v}
    def setName(v: String) {name = v}
    def setStand(v: String) {stand = v}
    def setCompany(v: String) {company = v}
    def setEmail(v: String) {email = v}
    def setAddress(v: String) {address = v}
    def setWebsite(v: String) {website = v}
    def setCategory(v: String) {category = v}
    def setLocation(v: String) {location = v}
    def setPhone(v: String) {phone = v}
    def setDescription(v: String) {description = v}
}
